lib/v2/yzw/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'yz.chsi.com.cn': {
```

Comment:
```suggestion
    'chsi.com.cn': {
```
The key should includes root domain only